### DISCUSSION TITLE
How to pass a parameter with command ?

### DISCUSSION THREAD

**Discussion by ilexpl01** [tobiasehlert/teslamateapi#246] [Q&A] [answered]
Hi,

Checked open & closed issues and discussion - could not find the answer (read: example).

Simple question:

How to pass 70% value in bellow command ?

curl -i -H "Authorization: Bearer -------------------------" -X POST "http://{URL}:8080/api/v1/cars/1/command/set_charge_limit 70"

The above does not work :/

BR to all :)

**Comment by ilexpl01** [comment, marked as answer]
After 2 hours:

curl -i -H "Authorization: Bearer -----------------------------------" -X POST "{URL}:8080/api/v1/cars/1/command/set_charge_limit" --data '{"percent" : "80"}'

other example (setting charging amp):

curl -i -H "Authorization: Bearer -------------------------------" -X POST "{URL}:8080/api/v1/cars/1/command/set_charging_amps" --data '{"charging_amps": "5"}'

No further questions ;P

**Comment by tobiasehlert** [comment]
Nice that you found it yourself @ilexpl01! ⭐ 

I don't have docs regarding commands itself, since it's just proxying commands as it is to the owner-api and the [tesla-api.timdorr.com](https://tesla-api.timdorr.com/vehicle/commands) docs are the one to look into for that.
